robot/basestation/app.py

```diff
@@ -170,8 +170,9 @@ def rover_drive():
 
 @app.route("/capture_image/", methods=["POST", "GET"])
 def capture_image():
-    stream_url = request.args['stream_url']
-    success, message = stream_capture(stream_url)
+    stream_url= request.args['stream_url']
+    rotation = request.args['camera_rotation']
+    success, message = stream_capture(stream_url, rotation)
```

Comment:
This isn't your fault because nobody has done it so far, but it would be a good time to document our functions. I suggest you document the methods. It is not clear what the "rotation" argument is. Is it 0,1,2,3, is it 0, 90, 180, 270 or is it "right", "left". Documenting the method would make this clear.